l2-frontend/src/forms/DescriptiveForm.vue

```diff
@@ -51,7 +54,7 @@
             >
               <div
                 v-if="field.title !== '' && !research.wide_headers"
-                class="field-title"
+                :class="[group.fieldsInline ? fieldTitleInline : 'field-title']"
```

Comment:
```suggestion
                :class="[group.fieldsInline ? 'fieldTitleInline' : 'field-title']"
```

Иначе это не класс, а переменная